### PR TITLE
Update sample details to latest spec

### PR DESCRIPTION
**Pull request by jjcastro**
Demo of this tested locally: 

Untrusted: https://screencast.googleplex.com/cast/NTEyMzkyOTg3NzI1MDA0OHw2NTdlMWViNS1jZQ

Trusted: https://screencast.googleplex.com/cast/NjcxNjMwODkyNzI4MzIwMHwzZWY5YzU0Ny1kMA

It's also been reorganised into small components that deal with each section of fraud/origin verification. This should make it easier for others to add sections for different parts of the origin verification report.

Note however the validity section is not connected to the backend yet. Everything else should be working.

It's also missing some strings for i18n, and some strings and colours which I don't know how they should look for trusted samples or for "improbable" / "impossible" validity.

### CODE REVIEW

File: sample_tracking/app/sample-details/components/DeforestationAlertsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { type Sample } from '../../utils';
+import { useTranslation } from 'react-i18next';
+
+type Props = {
```
Comment (benwulfe) [inline sample_tracking/app/sample-details/components/DeforestationAlertsSection.tsx:5]: Can we also add a comment here such as:
// Integration Point #1. Add the typescript snippet for your new section

File: sample_tracking/app/sample-details/page.tsx [hunk 2]
```diff
@@ -16,6 +16,13 @@ var QRCode = require('qrcode');
 import { QRCodeSVG } from "qrcode.react";
 import { useTranslation } from 'react-i18next';
 
+import ValidityTag from "./components/ValidityTag";
+import ValiditySection from './components/ValiditySection';
+import SampleOverviewSection from './components/SampleOverviewSection';
+import SampleDetailsSection from './components/SampleDetailsSection';
+import MeasurementsSection from './components/MeasurementsSection';
+import LandUseDetailsSection from './components/LandUseDetailsSection';
+import DeforestationAlertsSection from './components/DeforestationAlertsSection';
```
Comment (benwulfe) [inline sample_tracking/app/sample-details/page.tsx:25]: // Integration Point #2. Import your typescript snippet

File: sample_tracking/app/sample-details/page.tsx [hunk 3]
```diff
@@ -88,251 +95,43 @@ export default function SampleDetails() {
...
+                        <div className='header-validity'>
+                            <ValidityTag isTrusted={trusted === 'trusted'} city={selectedDoc['city']} lat={selectedDoc['lat']} lon={selectedDoc['lon']}/>
+                        </div>
+                        <div>
+                            {trusted != 'trusted' ? <ValiditySection selectedDoc={selectedDoc || {}} /> : ''}
+                            <SampleOverviewSection selectedDoc={selectedDoc || {}} />
+                            <SampleDetailsSection selectedDoc={selectedDoc || {}} sampleId={sampleId} />
+                            <MeasurementsSection selectedDoc={selectedDoc || {}} />
+                            {trusted != 'trusted' ? <LandUseDetailsSection selectedDoc={selectedDoc || {}} sampleId={sampleId} /> : ''}
+                            {trusted != 'trusted' ? <DeforestationAlertsSection selectedDoc={selectedDoc || {}}  /> : ''}
```
Comment (benwulfe) [inline sample_tracking/app/sample-details/page.tsx:123]: // Integration Point #3. display your new section in the desired order with respect to other snippets.

File: sample_tracking/app/sample-details/styles.css [hunk 4]
```diff
@@ -125,20 +139,24 @@ width: 200px;
...
 
 .legend.anthropic            { background-color: #fffbc2; }
 .legend.primary-vegetation   { background-color: #09611f; }
 .legend.secondary-vegetation { background-color: #4ea376; }
 
-.mapbiomas-footer {
-  margin-left: auto;
-  font-style: italic;
-  color: #5F6368;
+.alerts-source-footer {
```
Comment (benwulfe) [inline sample_tracking/app/sample-details/styles.css:153]: // Integration Point #4. Add any required styles for your section to the css.

**Review by benwulfe — CHANGES_REQUESTED** [review]
Really like how you split it out.
I added some suggestions per your original suggestion of marking integration points.
Otherwise, mostly comment nits

File: sample_tracking/app/utils.tsx [hunk 5]
```diff
@@ -17,6 +17,13 @@ export type UserData = {
   photoUrl: string,
 }
 
+export enum ValidityStatus {
+  Possible = 'Possible',
+  Trusted = 'Trusted',
+  Undetermined = 'Undetermined',
+  // TODO: add "impossible" or equivalent
```
[the remark this answers is not in the document]
Comment (jjcastro) [reply sample_tracking/app/utils.tsx]: Done, your interpretation is indeed what this implies

**Review by benwulfe — DISMISSED** [review]

**Review by benwulfe — APPROVED** [review]

**Pull request merged**